test/lib/sstable_utils.cc

```diff
@@ -177,8 +177,9 @@ future<compaction_result> compact_sstables(sstables::compaction_descriptor descr
     auto& cm = cf.get_compaction_manager();
```

Comment:
I don't see anything coroutinized here.

Reply:
sorry, it's a stale comment in the commitlog message.
I'll remove it.

Reply:
fixed in v7